src/celeritas/em/interactor/MuBetheBlochInteractor.hh

```diff
@@ -161,6 +161,9 @@ CELER_FUNCTION Interaction MuBetheBlochInteractor::operator()(Engine& rng)
 //---------------------------------------------------------------------------//
 /*!
  * Calculate maximum kinetic energy of the secondary electron.
+ *
+ * TODO this should be precomputed and used as part of the "applicability" per
+ * particle type, not included in the interactor. See celeritas#907 .
```

Comment:
How do we do that if it depends on the incident particle energy?

Reply:
The "applicability" for interactions should have a lower bound for the incident particle energy based on conservation rules etc. So, given a  particle type and the electron cutoff for a material, we should be able to solve for the lowest incident muon energy that creates an electron higher than the secondary cutoff. Right? The reason we can't do this now is because we don't have per-material applicability (https://github.com/celeritas-project/celeritas/issues/907).

Reply:
Ah yes, that's right. Actually, looking again I think the max secondary energy might already be used to limit the cross section/energy loss calculation in Geant4, in which case we shouldn't need to do anything except remove the (unnecessary) check in the interactor.